client/js/widgets/widget.js

```diff
@@ -672,7 +674,13 @@ export class Widget extends StateManaged {
             variables[variable] = getValue(variables[variable]);
           if(jeRoutineLogging) jeLoggingRoutineOperationSummary(a.substr(4), JSON.stringify(variables[variable]));
         } else {
-          problems.push('String could not be interpreted as expression. Please check your syntax and note that many characters have to be escaped.');
+          const comment = a.match(new RegExp('^ *(?://(.*))?\x24'));
```

Comment:
Since trim is used above, the RegExp does not need the leading ` *` to match leading spaces.

Reply:
Fixed.